### PR TITLE
feat: wrapped asset withdraw/borrow modal improvements

### PR DESCRIPTION
**Pull request by grothem**
## General Changes

- Adjusted borrow modal layout:
   - The Borrow APY rate switcher is now shown at the top
   - The unwrap toggle component is now outside of the Transaction overview section
   - On successful borrow of a wrapped asset, if the user borrows the asset unwrapped, don't show the 'Add token to wallet' section in the success modal.
- Adjusted withdraw modal layout:
   - The unwrap toggle component is now outside of the Transaction overview section
   - On successful withdraw of a wrapped asset, if the user withdraws the asset unwrapped, don't show the 'Add token to wallet' section in the success modal.
- Updated the fly-out menus for the 'View token contracts' and the 'Add token to wallet' on the reserve page. Added section titles and dividers.
- Updated token icons for Matic, Wrapped Matic and Wrapped AVAX
- On the reserve page actions, only show the Borrow action if the borrowing is enabled on the reserve. Previously, it would show for assets like Aave, but the button was disabled.
- Fixed a bug in the borrow modal, where if you changed the unwrap/wrap toggle after the transaction was built, the state of the toggle wasn't updating the transaction.

## Developer Notes

Add any notes here that may be helpful for reviewers.

---

## Author Checklist

Please ensure you, the author, have gone through this checklist to ensure there is an efficient workflow for the reviewers.

- [x]  The base branch is set to `main`
- [ ]  The title is using [Conventional Commit](https://www.conventionalcommits.org/en/v1.0.0/) formatting
- [ ]  The Github issue has been linked to the PR in the Development section
- [x]  The General Changes section has been filled out
- [ ]  Developer Notes have been added (optional)

**If the PR is ready for review:**

- [x]  The PR is in `Open` state and not in `Draft` mode
- [x]  The `Ready for Dev Review` label has been added

## Reviewer Checklist

Please ensure you, as the reviewer(s), have gone through this checklist to ensure that the code changes are ready to ship safely and to help mitigate any downstream issues that may occur.

- [ ]  End-to-end tests are passing without any errors
- [ ]  Code style generally follows existing patterns
- [ ]  Code changes do not significantly increase the application bundle size
- [ ]  If there are new 3rd-party packages, they do not introduce potential security threats
- [ ]  If there are new environment variables being added, they have been added to the `.env.example` file as well as the pertinant `.github/actions/*` files
- [ ]  There are no CI changes, or they have been approved by the DevOps and Engineering team(s)
- [ ]  Code changes have been quality checked in the ephemeral URL
- [ ]  QA verification has been completed
- [ ]  There are two or more approvals from the core team
- [ ]  Squash and merge has been checked

### CODE REVIEW

**Comment by grothem** [comment]
Link T-6742

**Review by iamanastasia — DISMISSED** [review]

File: src/modules/reserve-overview/TokenLinkDropdown.tsx [hunk 1]
```diff
@@ -66,25 +68,40 @@ export const TokenLinkDropdown = ({ poolReserve, downToSM }: TokenLinkDropdownPr
...
         >
           <TokenIcon symbol={poolReserve.iconSymbol} aToken={true} sx={{ fontSize: '20px' }} />
           <Typography variant="subheader1" sx={{ ml: 3 }} noWrap data-cy={`assetName`}>
             {'a' + poolReserve.symbol}
           </Typography>
         </MenuItem>
+        {showDebtTokenHeader && (
+          <Box sx={{ px: 4, pt: 3, pb: 2 }}>
+            <Typography variant="secondary12" color="text.secondary">
+              <Trans>Aave Debt Token</Trans>
```
Comment (defispartan) [inline src/modules/reserve-overview/TokenLinkDropdown.tsx:101]: Only the first word is capitalized in the first two headers, this one should probably be updated to match

**Comment by defispartan** [comment]
One other thing is in `src/components/icons/WalletIcon.tsx` there is <title> tag which causes the text "Wallet Icon" to appear when you hover this icon

File: src/modules/reserve-overview/TokenLinkDropdown.tsx (hunk 1 above)
Comment (grothem) [reply src/modules/reserve-overview/TokenLinkDropdown.tsx]: good catch, fixed

**Comment by grothem** [comment]
> One other thing is in `src/components/icons/WalletIcon.tsx` there is <title> tag which causes the text "Wallet Icon" to appear when you hover this icon

I removed the title element from the wallet and lens icons

**Review by defispartan — DISMISSED** [review]
LGTM

File: src/components/transactions/Borrow/BorrowActions.tsx [hunk 2]
```diff
@@ -43,7 +43,7 @@ export const BorrowActions = ({
         });
       },
       skip: !amountToBorrow || amountToBorrow === '0' || blocked,
-      deps: [amountToBorrow, interestRateMode],
+      deps: [amountToBorrow, interestRateMode, poolAddress],
```
Comment (defispartan) [inline src/components/transactions/Borrow/BorrowActions.tsx:46]: Good catch

**Review by defispartan — APPROVED** [review]

**Review by JoaquinBattilana — APPROVED** [review]

**Pull request merged**